apps/aecore/src/aec_chain_state.erl

```diff
@@ -42,6 +43,22 @@ get_n_key_headers_backward_from(Header, N) ->
     Node = wrap_header(Header),
     get_n_key_headers_from(Node, N).
 
+%% Compute the median timestamp for last aec_governance:median_timestamp_key_blocks()
+-spec median_timestamp(aec_headers:header()) -> {ok, integer()} | 'error'.
+median_timestamp(Header) ->
+    TimeStampKeyBlocks = aec_governance:median_timestamp_key_blocks(),
+    case aec_headers:height(Header) =< TimeStampKeyBlocks + 1 of
```

Comment:
So when validating block at height 12 this is not computing the median? This seems in conflict with [spec](https://github.com/aeternity/protocol/pull/167/files#diff-c423cd627c304e382e23fb48bcb356dfR143).